feat(scheduler-targets): add support for universal target (#32341)
aws/aws-cdk: packages/@aws-cdk/aws-scheduler-targets-alpha/README.md

  }),
});
```

## Invoke a wider set of AWS API

Use the `AwsApi` target to invoke AWS API.

The code snippet below creates an event rule with AWS API as the target which is
called at midnight every day by EventBridge Scheduler.

```ts
new Schedule(this, 'Schedule', {
  schedule: ScheduleExpression.cron({
    minute: '0',
    hour: '0',
  }),
  target: new targets.AwsApi({
    service: 'rds',
    action: 'stopDBCluster',
    input: ScheduleTargetInput.fromObject({
      DbClusterIdentifier: 'my-db',
    }),
  }),
});
```

> The `service` is must be in lower case and the `action` is must be in camelCase.

Author: My bad, fixed.
https://github.com/aws/aws-cdk/pull/32341/commits/209f6eac487c6724e79c287385f5a92908b43217

  }),
});
```

## Invoke a wider set of AWS API

Use the `Universal` target to invoke AWS API.

The code snippet below creates an event rule with AWS API as the target which is
called at midnight every day by EventBridge Scheduler.

```ts
new Schedule(this, 'Schedule', {
  schedule: ScheduleExpression.cron({
    minute: '0',
    hour: '0',
  }),
  target: new targets.Universal({
    service: 'rds',
    action: 'stopDBCluster',
    iamResources: ['arn:aws:rds:us-east-1:123456789012:cluster:my-cluster'],
    input: ScheduleTargetInput.fromObject({
      DbClusterIdentifier: 'my-db',
    }),
  }),
});
```

The `service` must be in lowercase and the `action` must be in camelCase.

> By default, IAM policies for the Scheduler are extracted from the API call.
> 
> You can also provide custom IAM policy statements to the Scheduler.
> 
> This is useful when you want to control the permissions of the Scheduler.

Reviewer: This would be better suited to the current props.

```suggestion
By default, an IAM policy for the Scheduler is extracted from the API call.

You can provide additional IAM policy statements to the Scheduler when
other permissions are required for the action.
```

Author: Updated README.
https://github.com/aws/aws-cdk/pull/32341/commits/2fe40a6aa1d3ed30f1247ae6db40180acf99bc2b
